### ISSUE TITLE
Autotracking is showing Powder when it hasn't been received.

### ISSUE THREAD

**Issue by TheT113** [StripesOO7/alttp-ap-poptracker-pack#6]
I didn't look into the code much to figure out the actual issue, but the autotracking ID for Powder seems to be wrong, as it's lighting up when I don't have Powder. Possibly linked to the wrong item.

**Comment by StripesOO7** [comment]
Yeah thats most likely because you did magic bat with fake powder. 
But yeah powder is a bit wonky and i will look over it.

**Comment by TheT113** [comment]
No, I did not. This issue was noticed because it showed Magic Bat (which is mislabeled "Satan" for some reason?) was green.

**Comment by StripesOO7** [comment]
Ah ok. Then i really need to look into that.
Yeah. I really should rename it to "Magic Bat" but the Satan comes from the group I usually play with 🫠 same with "brewing drugs" subtitle of the witch hut check.
